specification/electronic-prescription-service-api.yaml

```diff
@@ -489,6 +498,10 @@ paths:
           application/fhir+json:
             schema:
               type: object
+              description: |
+                The prescription orders to be verified, including the signatures,
+                expressed as a FHIR bundle of [prescription-order-update FHIR messages](https://simplifier.net/guide/NHSDigital-Medicines/Home/FHIRAssets/AllAssets/MessageDefinition-duplicate-2/prescription-order-update.guide.md).
```

Comment:
think [this](https://simplifier.net/guide/NHSDigital-Medicines/Home/FHIRAssets/AllAssets/Profiles/NHSDigital-Task.guide.md?version=current) link matches the other ones and is appropriate for `return` / `withdraw` interactions

Reply:
This is for the verify signature function, not for return or withdraw. So I don't think your comment is applicable.

Reply:
oops comment in wrong place apologies

Reply:
however, i think its inaccurate to say that the Bundle is a Bundle of `prescription-order-update` messages. the release response message that goes to this endpoint is a Bundle of prescriptions (`prescription-order` messages). think [this](https://simplifier.net/guide/NHSDigital-Medicines/Home/FHIRAssets/AllAssets/MessageDefinition-duplicate-2/prescription-order.guide.md) is the correct link

Reply:
Well spotted, sorry about that. Changed now, and for completeness I have added a similar description to the 200 response from the download prescriptions endpoint - which is essentially the same payload, I believe.